.github/workflows/scheduled-release.yml

```diff
@@ -0,0 +1,154 @@
+
+      - name: Get Latest Release
+        uses: octokit/request-action@v2.x
+        continue-on-error: true
+        id: get_latest_release_version
+        with:
+          route: GET /repos/acn-dgopa/altinn-access-management-frontend/releases/latest
+          mediaType: | # The | is significant!
+            format: raw
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: print status
+        run: "echo Release could not be found. Request failed with status '${{ steps.get_latest_release_version.outputs.status }}'"
+      
+      - name: split release version
+        if: ${{ steps.get_latest_release_version.outputs.status == '200' }}
+        uses: jungwinter/split@v2
+        id: split
+        with:
+          msg: "${{ fromJson(steps.get_latest_release_version.outputs.data).tag_name }}"
+          separator: '.'
+          
+      - name: split 'v' and year
+        if: ${{ steps.get_latest_release_version.outputs.status == '200' }}
+        uses: jungwinter/split@v2
+        id: splitv
+        with:
+          msg: "${{steps.split.outputs._0}}"
+          separator: 'v'
+          
+      - name: Increment release version
+        if: ${{ steps.get_latest_release_version.outputs.status == '200' }}
+        id: update_version
+        env:
+          NUM: ${{ steps.split.outputs._1 }}
+        run: |
+          echo "VERSION=$(($NUM + 1))" >> $GITHUB_ENV
+          
+      - name: Format new release version
+        if: ${{ steps.get_latest_release_version.outputs.status == '200' }}
+        id: format_version
+        env:
+          RELEASE_YEAR: ${{steps.splitv.outputs._1}}
+        run: |
+          if [ ${{env.currentyear}} -gt  ${{env.RELEASE_YEAR}} ]
+          then
+             echo "RELEASEVERSION=v${{env.currentyear}}.1" >> $GITHUB_OUTPUT
+          else
+             echo "RELEASEVERSION=v${{env.RELEASE_YEAR}}.${{env.VERSION}}" >> $GITHUB_OUTPUT
+          fi
+    
+      - name: print release version
+        id: releaseversion
+        run: |
+          if [ ${{ steps.get_latest_release_version.outputs.status }} = '200' ]
+          then
+            echo "RELEASE=${{ steps.format_version.outputs.RELEASEVERSION }}" >> $GITHUB_OUTPUT
+          else
+            echo "RELEASE=v${{env.currentyear}}.1" >> $GITHUB_OUTPUT
+          fi
+    outputs:
+      RELEASEVERSION: ${{ steps.releaseversion.outputs.RELEASE }}
+    
+  tag:
+    name: Tag image and commit
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+      issues: write
+      packages: write
+    needs: [getlatestrelease]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.COMMIT_ACCESS_TOKEN }}
+      - name: setup git config
+        run: |
+          # setup the username and email. I tend to use 'GitHub Actions Bot' with no email by default
+          git config user.name "dhanasneham@gmail.com"
```

Comment:
As the comment suggest maybe we should set a git user name which is recognizable as the "release github action bot"?